### PR TITLE
Increase compatibility with Jest 23.x

### PR DESCRIPTION
**Pull request by marcojahn**
- add [test|it].each
- add [test|it].only.each
- add [test|it].skip.each

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -74,7 +74,21 @@ function wrapTestInZone(testBody) {
   };
   if (methodName === 'test' || methodName === 'it') {
     env[methodName].only = env['fit'];
+    env[methodName].only.each = function (description, specDefinitions) {
+      arguments[1] = wrapTestInZone(specDefinitions);
```
Comment (thymikee) [inline index.js:78]: Have you checked what's inside `arguments`? `test.each` is a tagged template literal and its signature is rather `(strings, ...values) => (description, specDefinitions)`

Comment (marcojahn) [reply index.js:78]: I'm currently trying to get the right implementation done with this template literal

**Comment by AndersDJohnson** [comment]
We also need compatibility with `jest.each`.

**Comment by dzonatan** [comment]
Any news on this?

**Comment by thymikee** [comment]
PR still needs attention. Hopefully OP will find some time to iterate :)

**Pull request closed without merging**